src/modules/dcc/DccVideoWindow.cpp

```diff
@@ -654,11 +647,8 @@ void DccVideoWindow::connectionInProgress()
 const QString & DccVideoWindow::target()
 {
 	// This may change on the fly...
-	if(!m_pszTarget)
-		m_pszTarget = new QString();
-
-	m_pszTarget = QString::asprintf("%s@%s:%s", m_pDescriptor->szNick.toUtf8().data(), m_pDescriptor->szIp.toUtf8().data(), m_pDescriptor->szPort.toUtf8().data());
-	return *m_pszTarget;
+	m_szTarget = QString::asprintf("%s@%s:%s", m_pDescriptor->szNick.toUtf8().data(), m_pDescriptor->szIp.toUtf8().data(), m_pDescriptor->szPort.toUtf8().data());
```

Comment:
Pretty sure this should be
```c++
QString("%1@%2:%3").arg(m_pDescriptor->szNick).arg(m_pDescriptor->szIp).arg(m_pDescriptor->szPort);
```
to avoid extra charset conversions and for type safety.
But then I'll have to change all ~30 occurrences of `asprintf` for consistency, and that's beyond the scope of a simple build fix.